pipelines/cellpainting/cpd_max_projection_pipeline.wdl

```diff
@@ -61,25 +66,24 @@ workflow cpd_max_projection_distributed {
   }
 
   # Create new load_data/load_data_with_illum csv files with the new projected images
-  # and they are saved in the same folder
   call util.filter_csv as script {
     input:
-      full_load_data_csv= images_directory + "/load_data.csv",
-      full_load_data_with_illum_csv= images_directory + "/load_data_with_illum.csv",
+      full_load_data_csv= load_data_directory + "/load_data.csv",
+      full_load_data_with_illum_csv= load_data_directory + "/load_data_with_illum.csv",
```

Comment:
Oh....... I see.   So there's this `_with_illum` file too.  You could make this another concrete `String` input to the full file path?  I kind of think that's a more flexible way to go.

Reply:
Do you mean another `File` input?
Yes, I will add that, great suggestion.

Reply:
Yeah, if `File` is the right datatype, otherwise a `String`  or `String?` will probably work